### DISCUSSION TITLE
Carousel Block on  E-commerce > Category Preview delivers wrong component on the code

### DISCUSSION THREAD

**Discussion by ro93martinez** [primefaces/community#432] [PrimeVue Blocks] [answered]
![image](https://github.com/primefaces/community/assets/62414898/0d3ba3af-bee1-428f-a2c6-56f818605979)

The code I get to copy/paste makes reference to Galleria, it should by Carousel

**Comment by tugcekucukoglu** [comment, marked as answer]
Thanks for your attention. It will be fixed.
